packages/web/modals/validator-squad.tsx

```diff
@@ -0,0 +1,214 @@
+};
+
+interface ValidatorSquadContentProps {
+  onRequestClose: () => void;
+  isOpen: boolean;
+}
+
+const ValidatorSquadContent = observer(
+  ({ onRequestClose, isOpen }: ValidatorSquadContentProps) => {
+    const columnHelper = createColumnHelper<Validator>();
+
+    const virtualRows = data;
+    const rows = data;
+
+    const columns = useMemo(
+      () => [
+        columnHelper.accessor((row) => row, {
+          cell: "hello world",
+          header: "Validator",
+          id: "validator",
+          sortDescFirst: false,
+        }),
+        columnHelper.accessor((row) => row, {
+          cell: "hello world",
+          header: "My stake",
+          id: "myStake",
+          sortDescFirst: false,
+        }),
+        columnHelper.accessor((row) => row, {
+          cell: "hello world",
+          header: "Voting power",
+          id: "votingPower",
+          sortDescFirst: false,
+        }),
+        columnHelper.accessor((row) => row, {
+          cell: "hello world",
+          header: "Commissions",
+          id: "comissions",
+          sortDescFirst: false,
+        }),
+      ],
+      [columnHelper]
+    );
+
+    const table = useReactTable({
+      data,
+      // @ts-ignore
+      columns,
+      getCoreRowModel: getCoreRowModel(),
+      getSortedRowModel: getSortedRowModel(),
+      manualSorting: true,
+    });
+
+    const paddingTop = 0;
+    const topOffset = 324;
+
+    const handleSearchInput = () => console.log("search");
+
+    return isOpen ? (
+      <div className="absolute z-50 max-h-[938px] w-full max-w-[1168px] rounded-[20px] bg-osmoverse-800 px-[62px] pt-8 pb-[54px]">
+        <CloseButton
+          className="relative top-0 right-0"
+          onClick={() => onRequestClose()}
+        />
+        <div className="relative flex flex-col overflow-auto">
+          <div className="mx-auto mb-9 flex max-w-[500px] flex-col items-center justify-center">
+            <h6>Validator Squad</h6>
+            <div className="mt-7 mb-3 font-medium">
+              Select the validators you’d like to delegate to. Once complete,
+              continue to stake. You may edit your validator set at any time.
+            </div>
+            <SearchBox
+              placeholder="Search for a validator"
+              onInput={handleSearchInput}
+              className="self-end"
+              size="full"
+            />
+          </div>
+          <table className="w-full">
+            <thead className="z-[51] m-0">
+              {table.getHeaderGroups().map((headerGroup) => (
+                <tr key={headerGroup.id} className="!bg-osmoverse-700">
+                  {headerGroup.headers.map((header) => {
+                    return (
+                      <th key={header.id} colSpan={header.colSpan}>
+                        {header.isPlaceholder ? null : (
+                          <div
+                            {...{
+                              className: header.column.getCanSort()
```

Comment:
so, is each column sortable? And are we only able to evaluate one sort at a time?